npm/private/npm_translate_lock.bzl

```diff
@@ -16,6 +16,7 @@ DEFAULT_DEFS_BZL_FILENAME = "defs.bzl"
 
 _ATTRS = {
     "pnpm_lock": attr.label(),
+    "pnpm_workspace": attr.label(),
```

Comment:
Reminder to myself to add this in https://github.com/aspect-build/rules_js/pull/647.

Reply:
You should hold off on #647 as my refactoring here has grown rather large and will conflict with your changes.

Reply:
Once this is done & landed then #647 can be finished off